.circleci/pages-deploy.sh

```diff
@@ -0,0 +1,18 @@
+git config user.name "$USER_NAME"
+git config user.email "$USER_EMAIL"
+
+# Remove strict host checking for github.com
+mkdir -p ~/.ssh/
+echo -e "Host github.com\n\tStrictHostKeyChecking no\n\tUserKnownHostsFile /dev/null\n" >> ~/.ssh/config
+chmod 600 ~/.ssh/config
```

Comment:
It confuses me that this is necessary.  You'd think CircleCI could disable strict host checking for custom keys where domain is set.   It prevents the feature of being able to add your own private ssh keys unusable without this.